java/spacewalk-java.changes

```diff
@@ -1,3 +1,4 @@
+- Tech preview: add Single Sign On (SSO) via SAMLv2 protocol
```

Comment:
AFAIK we don't usually state if something is Tech preview or not at the changelog (can't find it for other tech previews we had), so I'd change this to:
```
- Add support for Single Sign On (SSO) via SAMLv2 protocol
```